api/modelmanager/modelmanager_test.go

```diff
@@ -403,6 +404,62 @@ func (s *modelmanagerSuite) TestModelStatusError(c *gc.C) {
 	c.Assert(out, gc.IsNil)
 }
 
+func (s *modelmanagerSuite) TestListModelsWithInfo(c *gc.C) {
+	userTag := names.NewUserTag("commander")
+	modelInfo := params.ModelInfo{
+		Name:               "name",
+		UUID:               "uuid",
+		ControllerUUID:     "controllerUUID",
+		ProviderType:       "aws",
+		DefaultSeries:      "xenial",
+		CloudTag:           "cloud-aws",
+		CloudRegion:        "us-east-1",
+		CloudCredentialTag: "credential-cred",
+		OwnerTag:           "user-admin",
+		Life:               params.Alive,
+		Status:             params.EntityStatus{Status: status.Status("active")},
+		Users:              []params.ModelUserInfo{},
```

Comment:
It would be odd to have a "commander" that has access to an "admin" model, but then isn't listed as one of the users of the model.
Then again, I suppose as long as the shape of the response is ok, we don't mind?

Reply:
we don't mind :)